src/storage/mvcc/txn.rs

```diff
@@ -551,6 +555,28 @@ impl<S: Snapshot> MvccTxn<S> {
         Ok(())
     }
 
+    /// Update a primary key's TTL if `advise_ttl > lock.ttl`.
+    ///
+    /// Returns the new TTL.
+    pub fn txn_heart_beat(&mut self, primary_key: Key, advise_ttl: u64) -> Result<u64> {
+        if let Some(mut lock) = self.reader.load_lock(&primary_key)? {
+            if lock.ts == self.start_ts {
+                if lock.ttl < advise_ttl {
+                    lock.ttl = advise_ttl;
+                    self.put_lock(primary_key, &lock);
+                    MVCC_TXN_HEART_BEAT_UPDATE_TTL.inc();
```

Comment:
I think it's similar to `grpc_msg_count`.

Reply:
Here I want to count only the requests that do update the lock's ttl. grcp_msg_count sounds like includes requests that didn't  update it.

Reply:
Yes. I think almost every request will update ttl...

Reply:
Oh, that make sense. So should I just remove the metric or add a metric for requests that didn't update ttl?

Reply:
What about logging the abnormality if advise_ttl < lock.ttl?

Reply:
I think we can leave some debug log for the case device_ttl<lock.ttl.

Reply:
Ok.